backend/migrations/00040_alter_programs.sql

```diff
@@ -0,0 +1,90 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE public.programs ADD COLUMN program_owner VARCHAR(255);
+ALTER TABLE public.programs ADD COLUMN funding_type VARCHAR(50);
+ALTER TABLE public.facilities_programs ADD COLUMN archived_at timestamp with time zone;
+ALTER TABLE public.program_sections ADD COLUMN capacity INT;
+ALTER TABLE public.program_sections ADD COLUMN name VARCHAR(255);
+ALTER TABLE public.program_sections ADD COLUMN instructor_name VARCHAR(255);
+ALTER TABLE public.program_sections ADD COLUMN description TEXT;
+ALTER TABLE public.program_sections ADD COLUMN archived_at timestamp with time zone;
+ALTER TABLE public.program_sections ADD COLUMN start_dt date;
+ALTER TABLE public.program_sections ADD COLUMN duration VARCHAR(32);
+ALTER TABLE public.program_sections ADD COLUMN total_hours INT;
+ALTER TABLE public.program_sections ADD COLUMN is_active boolean;
```

Comment:
How do this design allow for program sections to be 
- Scheduled
- Active
- Completed
- Cancelled
- Archived
- (eventually Paused?)

I think I can infer that:
- SCHEDULED (start_dt is in the future)
- ACTIVE (start_dt is in the past?)
- Not sure how to infer if something is completed or cancelled or paused.